lib/uptime_web.ex

```diff
@@ -60,6 +60,14 @@ defmodule UptimeWeb do
     end
   end
 
+  def component do
```

Comment:
we don't need to create a new one, non livecomponents are intended to use `html`

Reply:
Not sure what you mean, this makes it so I don't have to import the helpers everytime and it uses `use Phoenix.Component`

Reply:
you would do this `use UptimeWeb, :html` that function is defined in this file already